.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
@Soullivaneuh your autoloader doesn't fix anything (seldaek proposed a solution in https://github.com/composer/composer/issues/1493 to automatically call the ``__static`` method of every objects loaded which is not what you tried to fix in https://github.com/puli/cli/pull/21).
Whatever is the solution it is not this one, so should be reverted ;)